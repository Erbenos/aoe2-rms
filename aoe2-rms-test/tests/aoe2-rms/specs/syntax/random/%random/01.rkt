#lang aoe2-rms
; Place 5 or 6 or 7 gold mines with 6 being the most likely.

<OBJECTS-GENERATION>
(create-object 'GOLD
               (%random [30 (number-of-objects 5)]
                        [50 (number-of-objects 6)]
                        [20 (number-of-objects 7)]))
