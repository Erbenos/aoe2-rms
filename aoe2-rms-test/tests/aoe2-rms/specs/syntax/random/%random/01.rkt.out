<OBJECTS_GENERATION>
create_object GOLD {
  start_random
    percent_chance 30
      number_of_objects 5
    percent_chance 50
      number_of_objects 6
    percent_chance 20
      number_of_objects 7
  end_random
}
